experiments/openebs/openebs-nfs-provisioner-kill/openebs_nfs_provisioner_kill_ansible_logic.yml

```diff
@@ -0,0 +1,166 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    c_experiment: "openebs-nfs-provisioner-kill"
+    nfs_provisioner_ns: "{{ lookup('env', 'NFS_PROVISIONER_NAMESPACE')}}"
+    nfs_provisioner_label: "{{ lookup('env', 'NFS_PROVISIONER_LABEL')}}"
+    nfs_pvc: "{{ lookup('env', 'NFS_PVC') }}"
+    nfs_svc: "{{ lookup('env', 'NFS_SVC') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    c_lib: "{{ lookup('env', 'LIB') }}"
+    c_container: "{{ lookup('env','TARGET_CONTAINER') }}"
+    lib_image: "{{ lookup('env','LIB_IMAGE') }}"
+    c_engine: "{{ lookup('env','CHAOSENGINE') }}"
+    external_app_check: "{{ lookup('env', 'EXTERNAL_APP_CHECK') }}"
+    chaos_pod_name: "{{ lookup('env','POD_NAME') }}"
+    c_interval: "{{ lookup('env', 'CHAOS_INTERVAL')}}"
+    c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+
+  vars_files:
+    - /mnt/parameters.yml
+
+  tasks:
+
+    - block:
+      - include: openebs_nfs_provisioner_kill_prerequisites.yml
+
+      - include_vars:
+          file: chaosutil.yml
+
+      ## GENERATE EXP RESULT NAME
+      - name: Construct chaos result name (experiment_name)
+          set_fact:
+            c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
```

Comment:
c_result?

Reply:
> @rajdas98 Yes but that time you don't have total_chaos_duration env in the job so how your exp passed? because total_chaos_duration is a mandatory field for c_kill
> Can you please verify the restart count

Yes it is skipped because the condition c_duration is not defined. it is killing immediately